app/src/ui/tutorial/tutorial-panel.tsx

```diff
@@ -236,7 +237,8 @@ export class TutorialPanel extends React.Component<
           >
             <p className="description">
               Publishing will “push”, or upload, your commits to this branch of
-              your repository on GitHub.com.
+              your repository on GitHub.com. Publish using the third button in
```

Comment:
While I really like that we're using `GitHub.com` here it's possible that the repository will be published to a GitHub Enterprise Server and not to GitHub.com. I propose we make the simplest change possible here since we're coming up on a release and then we can open another low-risk PR that we can sneak into the production release which switches the language based on which endpoint we're publishing to.

```suggestion
              your repository on GitHub. Publish using the third button in
```

cc @billygriffin cause you'll have to approve this before the beta release.